MahApps.Metro/Styles/Controls.DataGrid.xaml

```diff
@@ -94,18 +94,21 @@
 
     <Style x:Key="MetroRowHeaderGripperStyle"
            TargetType="{x:Type Thumb}">
-        <Setter Property="Width"
-                Value="8" />
+        <Setter Property="Height"
+                Value="6" />
```

Comment:
ok, but the you should replace the `Rectangle` with the `Border` again, cause `Rectangle` must specify a width